### PR TITLE
Reformat and compact the description

### PR DESCRIPTION
**Pull request by harendra-kumar**
Second level bullets are not rendered properly so use a separate
paragraph instead to avoid the issue.

### CODE REVIEW

File: streamly.cabal [hunk 1]
```diff
@@ -3,33 +3,31 @@ name:               streamly
...
-  * Simple API, similar to standard Haskell lists.
-  * Declarative, auto scaled concurrency
-  * Simple, yet powerful abstractions
-      * Unifies streams, arrays and parsers
-      * Obviates the need for text and bytestring
-      * Covers the functionality of list-t and logict
-      * Covers all Data.List combinators
-      * Supports numerous time domain combinators
-      * Interworks with other streaming libraries
+  * Performance at par with C (<https://github.com/composewell/streaming-benchmarks Benchmarks>)
```
Comment (adithyaov) [inline streamly.cabal:12]:
```suggestion
  * Performance on par with C (<https://github.com/composewell/streaming-benchmarks Benchmarks>)
```
Does `at` also work here?

File: streamly.cabal [hunk 2]
```diff
@@ -3,33 +3,31 @@ name:               streamly
...
+  * Simple API, similar to standard Haskell lists (<https://github.com/composewell/streamly-examples Examples>)
+  * Declarative concurrency with automatic scaling
   * Filesystem, fsnotify, network and unicode support included
   * More functionality provided via many ecosystem packages
   .
-  Where to find more information:
+  Simple, unified and powerful abstractions:
```
Comment (adithyaov) [inline streamly.cabal:18]:
```suggestion
  Simple, unified, and powerful abstractions:
```

File: streamly.cabal [hunk 3]
```diff
@@ -3,33 +3,31 @@ name:               streamly
...
+  .
+      * Unifies streams, arrays, folds and parsers
```
Comment (adithyaov) [inline streamly.cabal:20]:
```suggestion
      * Unifies streams, arrays, folds, and parsers
```

File: streamly.cabal (hunk 1 above)
Comment (harendra-kumar) [reply streamly.cabal]: `on` is correct I think.

File: streamly.cabal (hunk 3 above)
Comment (harendra-kumar) [reply streamly.cabal]: done

File: streamly.cabal [hunk 4]
```diff
@@ -3,33 +3,31 @@ name:               streamly
...
+      * Unified streams and arrays obviate the need for text and bytestring
+      * Unifies Data.List, list-t and logict with streams
```
[the remark this answers is not in the document]
Comment (harendra-kumar) [reply streamly.cabal]: done

File: streamly.cabal (hunk 2 above)
Comment (harendra-kumar) [reply streamly.cabal]: done

**Pull request merged**